build/js/live-editor.ui.js

```diff
@@ -7,7 +7,7 @@ this["Handlebars"]["templates"]["tipbar"] = Handlebars.template(function (Handle
 function program1(depth0,data) {
   
   
-  return "x";}
+  return "&times;";}
```

Comment:
These are probably from a previous change where we forgot to include the compiled code.